runtime/dart_vm.cc

```diff
@@ -111,6 +111,15 @@ static const char* kDartTraceStreamsArgs[] = {
     "--timeline_streams=Compiler,Dart,Debugger,Embedder,GC,Isolate,VM,API",
 };
 
+static std::string DartOldGenHeapSizeArgs(uint64_t heap_size) {
+  static const int32_t buffer_size = 100;
```

Comment:
Use `constexpr` (though I believe the compiler will promote this to constexpr anyway)